tools/docker/syzbot/Dockerfile

```diff
@@ -83,6 +81,19 @@ RUN test "$(uname -m)" != x86_64 && exit 0 || \
     DEBIAN_FRONTEND=noninteractive apt-get install -t bookworm-backports -y -q \
     qemu-system-s390x qemu-system-mips qemu-system-ppc
 
+# Latest Rust compiler.
```

Comment:
This is copy-paste.

Why are we duplicating this? Is there a way to share Dockerfile somehow (include, derive)?

Reply:
I see Dockerfile.base is a common idiom to split out common stuff.

Reply:
Yes, the majority of `syzbot/Dockerfile` vs `env/Dockerfile` is unfortunately copy-paste. I think we could well be using just one, the only concern being its total size: currently both a ~6GB (which is already insane), yet some of their content is different, so the sum would be I guess ~8GB.

***

At least in this case I was hoping that there would be some sort of `include` statement to have common Dockerfile steps between the two, but I didn't find such a feature. It seems that the approach is to first build the common Docker container, then derive other containers from it.

Reply:
Just pointing it out - don't think this is actionable in this PR.